docs/API.md

```diff
@@ -89,6 +89,41 @@ mapProps(
 
 Accepts a function that maps owner props to a new collection of props that are passed to the base component.
 
+This allows props to be derived from other props in a similar way to [reselect](https://github.com/reactjs/reselect).
+```js
+const User = ({ name, status }) =>
```

Comment:
`mapProps` is simple for us. For someone new to the library, new to FP in general, and unable to read the type signature (extremely confusing without any experience writing TypeScript/Flow definition files), it's not straightforward at all. You can see that by looking at comments in issues since last year. :)

I'm sure I can cut down the example to 1/2 or 2/3 the size, though.

It would definitely be easier to organize this if there were a dedicated site rather than a markup file, but this is more of an intermediate solution.